packages/editor/src/client/blocks/richTextBlock.tsx

```diff
@@ -71,7 +74,9 @@ const TextTags: any = {
   I: () => ({[TextFormat.Italic]: true}),
```

Comment:
Some people also use `<em>` as a way to style italics, similarly to `<b>` and `<strong>`. So in order to stay coherent (you have `<b>` and `<strong>`) I would also include `<em>`.

Reply:
ok, thanks, will implement.

Reply:
italic and strong, both where options I did not actually change anything. But having a look again, they seem to be already quite the way you describe? .. If you see an issue there could you please commit yourself?

Reply:
ok, I see. No in that case I wouldn't change anything and leave the code as it is :)